synapse/tests/tools/test_dmon.py

```diff
@@ -0,0 +1,16 @@
+
+from synapse.tools.dmon import getArgParser
+import pytest
```

Comment:
Other than that, this looks great.  With those changes i'll mainline this PR today :)